app/views/administrators/bussiness.html.erb

```diff
@@ -7,16 +7,16 @@
       <thead>
         <tr class="bg-gray-500 text-white text-left">
           <th class="w-52 py-2"> Nombre </th>
-          <th class="w-28"> Rol </th>
+          <th class="w-52 py-2"> Usuario </th>
           <th class="w-28"> Categoría </th>
           <th class="w-28"> Productos Totales </th>
         </tr>
       </thead>
       <tbody>
         <% @companies.each do |company| %>
           <tr>
-            <td class="py-4"><%= company.name %></td>           
-            <td class="py-4"></td>
+            <td class="py-4"><%= company.name %></td> 
+            <td class="py-4"><%= @companies.find(company.id).user.email %></td>
             <td class="py-4 px-1"><%= @companies.find(company.id).category.name %></td>
             <td class="py-4 px-1"><%= Product.where(company_id: company.id).sum(:quantity)%> </td>
```

Comment:
avoid to add logic in your views, this should only show the data passed through the controller...

Reply:
Please pay attention to @dannharo 's feedback, since this logic should not appear in their views.